docs/docsite/rst/scenario_guides/guide_infoblox.rst

```diff
@@ -0,0 +1,304 @@
+.. _nios_guide:
+
+************************
+ Infoblox Guide
+************************
+
+.. contents:: Topics
+
+This guide describes how to use Ansible with the Infoblox Network Identity Operating System (NIOS).
+
+Introduction
+=============
+With Ansible integration, you can use Ansible playbooks to automate Infoblox Core Network Services for IP address management (IPAM), DNS, and inventory tracking.
+
+See the `Infoblox <https://www.infoblox.com/>`_ website for more information on the Infoblox product.
+
+NOTE: You can retrieve most of the example playbooks used in this guide from the  `network-automation/infoblox_ansible <https://github.com/network-automation/infoblox_ansible>`_ GitHub repository.
+
+Prerequisites
+-------------
+Before using Ansible ``nios`` modules with Infoblox, you must install the ``infoblox-client`` on your Ansible control node:
+
+.. code-block:: bash
+
+    $ sudo pip install infoblox-client
+
+Credentials and authenticating
+==============================
+
+To use Infoblox ``nios`` modules in playbooks, you need to configure the credentials to access your Infoblox system.  The examples in this guide use credentials stored in ``<playbookdir>/group_vars/nios.yml``.
+
+This is an example of this ``group_vars/nios.yml`` file. Replace these values with your Infoblox credentials:
+
+.. code-block:: yaml
+
+    ---
+    nios_provider:
+      host: 192.0.0.2
+      username: admin
+      password: ansible
+
+Module list
+============
+
+See :ref:`nios_net tools_modules` for a full list of NIOS modules for this release.
+Each module includes simple documented example tasks for how to use them. `Use cases with modules`_ expands on some of these example tasks.
+
+NIOS lookup plugins
+===================
+
+Ansible includes the following lookup plugins for NIOS:
+
+- :ref:`nios <nios_lookup>` Uses the Infoblox WAPI API to fetch NIOS specified objects, for example network views, DNS views, and host records.
+- :ref:`nios_next_ip <nios_next_ip_lookup>` Provides the next available IP address from a network. You'll see an example of this in `Creating a host record`_.
+- :ref:`nios_next_network <nios_next_network_lookup>` - Returns the next available network range for a network-container.
+
+.. note:: You must run these lookup plugins locally by specifying ``connection: local``.
+
+
+Retrieving all network views
+----------------------------
+
+This example playbook uses the ``set_fact`` module with the ``nios`` lookup plugin to retrieve all the network views, which are then saved in the ``networkviews`` variable:
```

Comment:
I think "this example playbook" is fluff - how about rewording these examples using the pattern: To do X, use Y:  <example_playbook>? So for this one:
```
To retrieve all network views and save them in a variable, use the `set_fact` module with the `nios` lookup plugin:
```